.github/workflows/ci.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       - name: Maven Checks
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          $RETRY $MAVEN clean install -B --strict-checksums -V -T C1 -DskipTests -P ci -pl '!:trino-server-rpm'
+          $RETRY $MAVEN clean package -B --strict-checksums -V -T C1 -DskipTests -P ci -pl '!:trino-server-rpm'
```

Comment:
`verify` is the stage just before `install`. `package` is even earlier.

Can we somehow get list of which plugins are bound to `install` phase to make sure nothing is skipped?

Reply:
WDYM by bound to?

Reply:
Plugins execute based on what lifecycle phase they are "bound" to. e.g. maven-compiler plugin is bound to "compile" phase, surefire is bound to "test" phase, failsafe plugin to "integration-test" phase.

I know I can look at pom.xml and see the `<phase>` under `<execution>` for a plugin but not sure if we can get a list from Maven which includes inherited stuff as well.

Reply:
Oh, Maven plugins, all clear now. I'll replace `package` with `verify` everywhere, that should be much safer. I think I remember reading about some Maven commands to list all plugins and phases, I'll look it up.

Reply:
Actually, I used `package` instead of `verify` everywhere we also add `-DskipTests`. But I'll check the plugins anyway.

Reply:
I ran this `./mvnw fr.jcgay.maven.plugins:buildplan-maven-plugin:list-phase` and for `install` there's only `maven-install-plugin` for all modules. `verify` has `spotbugs-maven-plugin` and `maven-pmd-plugin`, so we need to run it instead of just `package`.

One example for `trino-hive`:
```
[INFO] Build Plan for trino-hive: 
validate -----------------------------------------------------------------------
    + maven-enforcer-plugin         | default               | enforce          
    + dependency-scope-maven-plugin | default               | check            
    + license-maven-plugin          | default               | check            
    + maven-checkstyle-plugin       | checkstyle            | check            
initialize ---------------------------------------------------------------------
    + git-commit-id-plugin          | default               | revision         
    + jacoco-maven-plugin           | default               | prepare-agent    
process-resources --------------------------------------------------------------
    + maven-resources-plugin        | default-resources     | resources        
compile ------------------------------------------------------------------------
    + maven-compiler-plugin         | default-compile       | compile          
process-test-resources ---------------------------------------------------------
    + maven-resources-plugin        | default-testResources | testResources    
test-compile -------------------------------------------------------------------
    + maven-compiler-plugin         | default-testCompile   | testCompile      
process-test-classes -----------------------------------------------------------
    + maven-dependency-plugin       | default               | analyze-only     
    + maven-dependency-plugin       | default               | analyze-duplicate
    + duplicate-finder-maven-plugin | default               | check            
    + modernizer-maven-plugin       | modernizer            | modernizer       
test ---------------------------------------------------------------------------
    + maven-surefire-plugin         | default-test          | test             
prepare-package ----------------------------------------------------------------
    + jacoco-maven-plugin           | report                | report           
package ------------------------------------------------------------------------
    + maven-jar-plugin              | default-jar           | jar              
    + maven-jar-plugin              | attach-tests          | test-jar         
    + maven-source-plugin           | attach-sources        | jar-no-fork      
    + maven-source-plugin           | attach-sources        | test-jar-no-fork 
verify -------------------------------------------------------------------------
    + spotbugs-maven-plugin         | default               | check            
    + maven-pmd-plugin              | default               | check            
install ------------------------------------------------------------------------
    + maven-install-plugin          | default-install       | install          
deploy -------------------------------------------------------------------------
    + maven-deploy-plugin           | default-deploy        | deploy        
```